red_black_cards.py

```diff
@@ -0,0 +1,44 @@
+#Problem Statement: "You are given a standard deck of cards with 26 black cards and 26 red cards. You're now going to play a game: 
+#in every round, you can take one of two actions: you can stop playing, or you can draw another card. If you draw another card and 
+#it's red, you get one dollar. If you draw another card and it's black, you lose one dollar. What is the expected value of this game?"
```

Comment:
Style guide is to use a space after `#`. Please install `black`, the linter, as per [here](https://docs.google.com/document/d/1p6A8qcyQ89HTHo8M684AE4bDBhjAHZFZwB0iGdoD6Hg/edit), which will automatically catch this in your IDE (I suggest Pycharm)